cmd/flags.go

```diff
@@ -112,13 +112,19 @@ var (
 	// P2PUDPPort defines the port to be used by discv5.
 	P2PUDPPort = &cli.IntFlag{
 		Name:  "p2p-udp-port",
-		Usage: "The port used by discv5.",
+		Usage: "The UDP port used by the discovery service discv5.",
 		Value: 12000,
 	}
-	// P2PTCPPort defines the port to be used by libp2p.
+	// P2PQUICPort defines the QUIC port to be used by libp2p.
+	P2PQUICPort = &cli.IntFlag{
```

Comment:
Along with this flag, is it possible to also add a feature flag `enable-quic` ? We do not want QUIC to immediately be the default as if there are issues with other Lighthouse nodes and their QUIC implementation it could lead to issues for node operators.

Reply:
Fixed in https://github.com/prysmaticlabs/prysm/pull/13786/commits/c69efdb8d9b2da87ce224b0f71f50929a4cdd71c.